plugins/foreach.yaml

```
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: foreach
+spec:
+  version: v0.1.0
+  homepage: https://github.com/ahmetb/kubectl-foreach
+  shortDescription: Run kubectl commands against some/all contexts in parallel
+  description: |
+    Lets you run the same kubectl command against multiple contexts
+    simultaneously and prints their output, prefixed by context name.
+    You can choose or exclude contexts with exact name match or regular
+    expression patterns.
```

```suggestion
    Run the same kubectl command against multiple contexts
    simultaneously and print their output, prefixed by context name.
    Choose or exclude contexts with exact name match or regular
    expressions.
```
minor nit but something written in the imperative sounds better imo, wdyt?